__tests__/endpoints.test.js

```diff
@@ -66,9 +66,17 @@ describe("GET requests", () => {
             topic: "mitch",
             created_at: "2020-07-09T20:11:00.000Z",
             votes: 100,
+            comment_count: "11",
```

Comment:
you shouldn't be updating old tests, add new tests for new functionality (ensuring the old test still pass)